manifests/concourse-manifest/extensions/datadog-agent.yml

```diff
@@ -1,8 +1,8 @@
 releases:
   - name: riemann
-    url: https://github.com/alphagov/paas-cf/releases/download/riemann-1/riemann-gds-1.tgz
-    sha1: 8a4220a039ce6a2a0383aab8d4a76746501e58ec
-    version: gds-1
+    url: https://s3-eu-west-1.amazonaws.com/gds-dev-boshreleases/riemann-gds-4.tgz
```

Comment:
The commit message and the code give me no indication where this release came from. 

It is stored in a bucket I do not recognise. Regarding the bucket:
    * Why is the prefix `gds-` not `gds-paas-`?
    * How was this bucket created?
    * Is this bucket going to be there permanently?

I can see from the pull request notes it came from a fork we have created (paas-riemann-boshrelease). Why did we not serve the `tgz` as a GitHub release under that repository?